python/pyarrow/parquet/__init__.py

```diff
@@ -3114,8 +3114,11 @@ def file_visitor(written_file):
         the entire directory will be deleted.  This allows you to overwrite
         old partitions completely.
     **kwargs : dict,
-        Additional kwargs for write_table function. See docstring for
-        `write_table` or `ParquetWriter` for more information.
+        When `use_legacy_dataset` is False, used as additional kwargs for 
+        `pyarrow.dataset.ParquetFileFormat.make_write_options` function.
```

Comment:
I think this should be ...

```suggestion
        `pyarrow.dataset.write_dataset` function.
```
Then something like this can be added: (See docstring for `write_dataset` or  `ParquetFileFormat` for more information).

Reply:
Done.